addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1607,14 +1607,16 @@ export class WeInput extends UserValueComponent {
     };
     static defaultProps = {
         unit: "",
-        step: "1",
     };
     static components = {};
     static StateModel = UnitUserValue;
 
     setup() {
         super.setup();
         this.inputRef = useRef("input");
+        if (this.state._isNumeric() && !this.props.step) {
+            this.props.step = "1";
+        }
```

Comment:
You cannot modify a prop, you should either handle it in the UserValue or have a new variable containing the new infos.

cc @robinlej I assume since this is your commit, that it comes for your branch and was just cherry-picked here so this comment is more for you 😄